src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,11 +30,38 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
+
   // The robot's subsystems are defined here...
   private final Drivetrain m_drive = new Drivetrain();
 
   // Shuffleboard stuff
   SendableChooser<Command> m_autoCommand = new SendableChooser<>();
+  SendableChooser<Teams> m_teamChooser = new SendableChooser<>();
+
+  // Vision stuff 
+  public static Node selectedNode = null;
+
+  /// Selection values (grid, row, spot)
+  public static int[] selectValues = {0,0,0};
+
+  // Timer for clearing array
+  public static double selectTime;
+
+  // How much time it should take (in frames)
+  public final static double selectTimeAmount=100;
+
+  // Array of april tags. The index of the april tag in the array is equal to its id, and aprilTags[0] is null.
+  public final static Pose3d[] aprilTags = new Pose3d[9];
+
+  // 2D arrays of nodes. blueNodes[3][1] will return the top row cone node on the far left side (from the perspective of the driver)
+  public final static Node[][] blueNodes = new Node[4][];
+  public final static Node[][] redNodes = new Node[4][];
+
+
+
+  // Possible teams
+  public static enum Teams {BLUE, RED};
+  public static Teams team;
```

Comment:
You can use Alliance.Red or Alliance.Blue, that is a already included WPI class that the driverstation class uses.